app/views/papers/_sort_by.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="btn-group btn-group-xs nav-button-group sort" role="group" aria-label="...">
+  <%= link_to 'Votes', page == 'list' ? list_path(sort: 'votes') : papers_path(sort: 'votes'), class: "btn btn-default" + (params[:sort] == "pub_date" ? "" : " active"), role: "button" %>
+  <%= link_to 'Publish Date', page == 'list' ? list_path(sort: 'pub_date') : papers_path(sort: 'pub_date'), class: "btn btn-default" + (params[:sort] == "pub_date" ? " active" : ""), role: "button"  %>
```

Comment:
Ternary operators are great, but this is really hard to read. Also, `page == 'list'` seems like a code smell to me. I don't like it.